lib/actual_db_schema/patches/migration_context.rb

```diff
@@ -69,21 +69,13 @@ def migrate(migration)
         migrator.migrate
       end
 
-      def db_config
-        @db_config ||= if ActiveRecord::Base.respond_to?(:connection_db_config)
-                         ActiveRecord::Base.connection_db_config.configuration_hash
-                       else
-                         ActiveRecord::Base.connection_config
-                       end
+      def metadata
+        @metadata ||= ActualDbSchema::Store.instance.read
       end
 
       def branch_for(version)
         metadata.fetch(version, {})[:branch] || "unknown"
       end
-
-      def metadata
```

Comment:
Can you please move these lines back so that we don't have the not relevant diff in git?